data/json/items/melee/swords_and_blades.json

```
@@ -1,4 +1,35 @@
 [
+  {
+    "id": "sword_wood",
+    "type": "GENERIC",
+    "category": "weapons",
+    "weapon_category": [ "1H_SWORDS" ],
+    "symbol": "!",
+    "color": "brown",
+    "name": { "str": "2-by-sword" },
+    "description": "A two by four with a cross guard and whittled down point; not much for slashing, but a sharp point is better than nothing.",
+    "material": "wood",
+    "volume": "1250 ml",
+    "weight": "600 g",
+    "bashing": 4,
+    "cutting": 10,
+    "to_hit": 1,
+    "price_postapoc": 50,
+    "flags": [ "SHEATH_SWORD", "STAB" ],
+    "techniques": [ "WBLOCK_1" ]
+  },
+  {
+    "id": "sword_nail",
+    "type": "GENERIC",
+    "copy-from": "sword_wood",
+    "color": "light_gray",
+    "name": { "str": "nail sword" },
+    "description": "The nail sword, or nord for short.  This wooden sword has a dozen nails sticking out at jagged angles from the edge of the blade, adding a bit of piercing to its swings.",
+    "volume": "1750 ml",
+    "weight": "648 g",
```

```suggestion
    "weight": "650 g",
```
i get that 4g(nail) * 12 = 48g but maybe it could be rounded to be more visually pleasing